development/applications/hedwig/src/flight_control.c

```diff
@@ -298,6 +300,26 @@ static void flight_control_reset(void)
   }  
 }
 
+void flight_control_calibrate_sensors(void)
+{
+  sensors_calibrate_cmd = true;
```

Comment:
I don't love this name I would probably make it something like `calibration_required` and I also would name the function `flight_control_calibrate_cmd_rx` and do the packet checking here

Reply:
maybe don't do the second part of that - I'm not sure that, that is better